core/src/main/java/org/mule/construct/AbstractPipeline.java

```diff
@@ -91,6 +90,24 @@ protected MessageProcessor createPipeline() throws MuleException
         return builder.build();
     }
 
+    protected ProcessingStrategy createDefaultProcessingStrategy()
```

Comment:
Default can't only go in MuleConfiguration?  Or is this because of need to test without MueContext etc.?